### PR TITLE
Add zero opacity to Box

### PR DESCRIPTION
**Pull request by michaeltaranto**

### CODE REVIEW

File: lib/components/Toggle/Toggle.treat.ts [hunk 1]
```diff
@@ -16,14 +16,10 @@ export const root = style({
 });
 
 const realFieldBase = style({
-  opacity: 0,
   height: hitArea,
   zIndex: 1,
   selectors: {
     ...debugTouchable(),
-    [`&:not(:disabled)`]: {
```
Comment (markdalgleish) [inline lib/components/Toggle/Toggle.treat.ts:24]: Why was this removed?

File: lib/components/Toggle/Toggle.treat.ts [hunk 2]
```diff
@@ -37,24 +33,15 @@ const realFieldPosition = style((theme) => {
 
 export const realField = [realFieldBase, realFieldPosition];
 
-export const label = [
-  style({
-    selectors: {
-      [`${realFieldBase}:not(:disabled) ~ &`]: {
-        cursor: 'pointer',
```
Comment (markdalgleish) [inline lib/components/Toggle/Toggle.treat.ts:44]: Why was this removed? (Part 2)

File: lib/components/Toggle/Toggle.treat.ts (hunk 1 above)
Comment (michaeltaranto) [reply lib/components/Toggle/Toggle.treat.ts:24]: A toggle does not support being disabled. This was a copy and paste from `InlineField` originally i believe. So moved the `cursor: pointer` to box props (same for your next comment)

File: lib/components/Toggle/Toggle.tsx [hunk 3]
```diff
@@ -48,6 +48,8 @@ export const Toggle = ({
         checked={on}
         onChange={handleChange(onChange)}
         position="absolute"
+        cursor="pointer"
```
Comment (markdalgleish) [inline lib/components/Toggle/Toggle.tsx:51]: Ah, just saw this. Does this still apply when it's disabled?

Comment (michaeltaranto) [reply lib/components/Toggle/Toggle.tsx:51]: You can't disable a toggle

Comment (markdalgleish) [reply lib/components/Toggle/Toggle.tsx:51]: Didn't catch your earlier comment. Please ignore me.

File: lib/components/Toggle/Toggle.treat.ts (hunk 1 above)
Comment (markdalgleish) [reply lib/components/Toggle/Toggle.treat.ts:24]: Nice clean up 👍

File: .changeset/dry-forks-relate.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+'braid-design-system': minor
+---
+
+Add zero opacity to Box
+
+Provide zero opacity on `Box` as an optimisation.
```
Comment (markdalgleish) [inline .changeset/dry-forks-relate.md:7]: Probably worth adding an example usage here. Would also help make it clear to consumers that it's a number.
